Sources/Jetworking/Client/Client.swift

```diff
@@ -351,6 +362,7 @@ extension Client: DownloadExecutorDelegate {
     public func downloadExecutor(_ downloadTask: URLSessionDownloadTask, didFinishDownloadingTo location: URL) {
         // TODO handle response before calling the completion
         guard let completionHandler = executingDownloads[downloadTask.identifier]?.completionHandler else { return }
+        sessionCache.store(location, from: downloadTask)
```

Comment:
According to the documentation of `URLSessionDownloadDelegate` we can't be sure how long the file at `location` exists:
```
**location**
A file URL for the temporary file. Because the file is temporary, you must either open the file for reading or move it to a permanent location in your app’s sandbox container directory before returning from this delegate method.
```

We should either move the file to a permanent location or check in `download` method if the file still exists prior to call the completion.

Reply:
Done